ios/AppLovinMAX.m

```diff
@@ -206,6 +206,128 @@ - (instancetype)init
     }];
 }
 
+#pragma mark - Data Passing
+
+RCT_EXPORT_METHOD(setTargetingDataYearOfBirth:(nonnull NSNumber *)yearOfBirth)
+{
+    if ( !_sdk )
+    {
+        [self logUninitializedAccessError: @"setTargetingDataYearOfBirth"];
+        return;
+    }
+    
+    self.sdk.targetingData.yearOfBirth = yearOfBirth.intValue <= 0 ? nil : yearOfBirth;
+}
+
+RCT_EXPORT_METHOD(setTargetingDataGender:(NSString *)gender)
+{
+    if ( !_sdk )
+    {
+        [self logUninitializedAccessError: @"setTargetingDataGender"];
+        return;
+    }
+    
+    ALGender alGender = ALGenderUnknown;
+
+    if ( [@"F" isEqualToString: gender] )
+    {
+        alGender =  ALGenderFemale;
+    }
+    else if ( [@"M" isEqualToString: gender] )
+    {
+        alGender =  ALGenderMale;
+    }
+    else if ( [@"O" isEqualToString: gender] )
+    {
+        alGender =  ALGenderOther;
+    }
+
+    self.sdk.targetingData.gender = alGender;
+}
+
+RCT_EXPORT_METHOD(setTargetingDataMaximumAdContentRating:(nonnull NSNumber *)maximumAdContentRating)
+{
+    if ( !_sdk )
+    {
+        [self logUninitializedAccessError: @"setTargetingDataMaximumAdContentRating"];
+        return;
+    }
+    
+    ALAdContentRating rating = ALAdContentRatingNone;
+
+    int intVal = [ maximumAdContentRating intValue ];
+
+    if ( intVal == 1 )
+    {
+        rating = ALAdContentRatingAllAudiences;
+    }
+    else if ( intVal == 2 )
+    {
+        rating = ALAdContentRatingEveryoneOverTwelve;
+    }
+    else if ( intVal == 3 )
+    {
+        rating = ALAdContentRatingMatureAudiences;
+    }
+
+    self.sdk.targetingData.maximumAdContentRating = rating;
+}
+
+RCT_EXPORT_METHOD(setTargetingDataEmail:(NSString *)email)
```

Comment:
We might need to let the pub be able to clear this (and other) values. 

Our iOS SDK API:

```
/**
 * The email of the user.
 * Set this property to @c nil to clear this value.
 */
@property (nonatomic, copy, nullable) NSString *email;
```

Reply:
Ok, I'll add 'nullable' for those functions explicitly.